.github/workflows/swift_package_test.yml

```diff
@@ -511,8 +511,7 @@ jobs:
       fail-fast: false
       matrix:
         swift_version: ${{ fromJson(inputs.android_sdk_versions) }}
-        sdk_triple: ${{ fromJson(inputs.android_sdk_triple) }}
```

Comment:
Question about removing the triple from the matrix axis: does this make it harder for end users to diagnose build issues in a particular architecture?

Just thinking from the concept of a CI job, if a dev were testing their package and wanted to test against the Swift SDK for Android, I would think it would create a job run per triple, but admittedly I haven't though too deeply about it: So I guess the outstanding question is just what the benefit is from a CI users perspective

Reply:
makes sense, thanks for explaining 👍

Reply:
This changes the default from x86_64 only to x86_64 + aarch64, and the overhead will of course be proportional to the package size. Mainly I wanted to ensure the multiarch support worked correctly in testing this PR, but I think it's beneficial to have both on by default since aarch64 is what's used for the vast majority of devices, and x86_64 for the majority of at-desk testing in the emulator.

Could also revert to a single arch if you strongly prefer.